google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1756,31 +1772,90 @@ def default_acl
         #
         # @return [Policy] the current Cloud IAM Policy for this bucket
         #
-        # @example
+        # @example Retrieving a Policy that is implicitly version 1:
         #   require "google/cloud/storage"
         #
         #   storage = Google::Cloud::Storage.new
-        #
         #   bucket = storage.bucket "my-todo-app"
         #
         #   policy = bucket.policy
+        #   policy.version # 1
+        #   puts policy.roles["roles/storage.objectViewer"]
         #
-        # @example Retrieve the latest policy and update it in a block:
+        # @example Retrieving a version 3 Policy using `requested_policy_version`:
         #   require "google/cloud/storage"
         #
         #   storage = Google::Cloud::Storage.new
+        #   bucket = storage.bucket "my-todo-app"
+        #
+        #   policy = bucket.policy requested_policy_version: 3
+        #   policy.version # 3
+        #   puts policy.bindings.find do |b|
+        #     b[:role] == "roles/storage.objectViewer"
+        #   end
         #
+        # @example Updating a Policy that is implicitly version 1:
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
         #   bucket = storage.bucket "my-todo-app"
         #
         #   bucket.policy do |p|
-        #     p.add "roles/owner", "user:owner@example.com"
+        #     p.version # 1
```

Comment:
Instead of highlighting only `1`, could you instead state "the value is `1`".

Reply:
A common Ruby syntax for this is:
```ruby
p.version # => 1
```

Reply:
@dazuma Yes, it's frustrating not to be able to use that notation. yard-doctest dynamically generates equals assertions for the value to the right of the hash rocket and evaluates them. Unfortunately, the evaluation always errors when generated from code within a block.

```sh
Google::Cloud::Storage::Bucket#policy#test_0001_Updating a Policy that is implicitly version 1::
SyntaxError: /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:108: syntax error, unexpected end-of-input, expecting end
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:98:in `eval'
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:98:in `evaluate'
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:92:in `evaluate_with_assertion'
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:75:in `assert_example'
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:54:in `block (4 levels) in generate'
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:49:in `each'
    /Users/quartzmo/.rbenv/versions/2.6.5/gemsets/quartzmo/gems/yard-doctest-0.1.17/lib/yard/doctest/example.rb:49:in `block (3 levels) in generate'
```

@blowmage and I settled on just using `#` without the hash rocket in these cases.